scripts/lib/upgrade-zulip-stage-2

```diff
@@ -82,6 +87,25 @@ try:
 except (configparser.NoSectionError, configparser.NoOptionError):
     tornado_processes = 1
 
+# postgresql.version is effectively required, but wasn't previously;
+# fill it in based on what the OS provides.
+try:
+    postgres_version = int(config_file.get('postgresql', 'version'))
+except (configparser.NoSectionError, configparser.NoOptionError):
```

Comment:
It's probably worth extracting the "read a value" to a function in `zulip_tools.py` that we can call.  Oh, we already did, `get_config`.  We can use that here, and maybe add a function for doing the "set, creating section if needed" use case too.